Norm_end_to_end.Rmd

```diff
@@ -432,200 +524,450 @@ colnames(study_summary) <- colnames(study_sample)
+abline(h = 0, lty=3, lwd = 1, col = "black")
+
+vioplot(log2(assay(res, "NOMIS")),
+        cex = 0.5, col = col_sample, ylab = expression(log[2]~abundance),
+        border = paste0(col_sample, 40), xaxt = "n",
+        main = "NOMIS", cex.main = 1)
+grid(nx = NA, ny = NULL)
+abline(h = 0, lty=3, lwd = 1, col = "black")
+legend("bottomleft", inset = c(0, -0.2), col = col_phenotype,
+       legend = names(col_phenotype), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+       cex = 0.7)
+```
+
+
+### PCA for each
+
+Median scaled
+
+```{r, include=TRUE, fig.width = 10, fig.height = 5}
+vals <- apply(assay(res, "median_filled"), MARGIN = 1, na.unidis) |>
+    log2() |>
+    scale(center = TRUE, scale = TRUE)
+
+pca_res <- prcomp(vals, scale = FALSE, center = FALSE)
+
+par(mfrow = c(1, 2), mar = c(4.5, 4.5, 4.5, 1))
+plot_pca(pca_res, pc_x = 1, pc_y = 2, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index, main = "median scaled")
+legend("topleft", inset = c(0, -0.24), col = col_phenotype,
+       legend = names(col_phenotype), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+       cex = 0.7, title = "Phenotype", title.cex = 0.8)
+plot_pca(pca_res, pc_x = 3, pc_y = 4, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
+```
+
+Median scaled 500
+
+```{r, include=TRUE, fig.width = 10, fig.height = 5}
+vals <- apply(assay(res, "median_filled_500"), MARGIN = 1, na.unidis) |>
+    log2() |>
+    scale(center = TRUE, scale = TRUE)
+
+pca_res <- prcomp(vals, scale = FALSE, center = FALSE)
+
+par(mfrow = c(1, 2), mar = c(4.5, 4.5, 4.5, 1))
+plot_pca(pca_res, pc_x = 1, pc_y = 2, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index, main = "median scaled 500")
+legend("topleft", inset = c(0, -0.24), col = col_phenotype,
+       legend = names(col_phenotype), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+       cex = 0.7, title = "Phenotype", title.cex = 0.8)
+plot_pca(pca_res, pc_x = 3, pc_y = 4, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
+```
+
+RUVR
+
+```{r, include=TRUE, fig.width = 10, fig.height = 5}
+vals <- apply(assay(res, "ruvr"), MARGIN = 1, na.unidis) |>
+    log2() |>
+    scale(center = TRUE, scale = TRUE)
+
+pca_res <- prcomp(vals, scale = FALSE, center = FALSE)
+
+par(mfrow = c(1, 2), mar = c(4.5, 4.5, 4.5, 1))
+plot_pca(pca_res, pc_x = 1, pc_y = 2, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index, main = "RUVR")
+legend("topleft", inset = c(0, -0.24), col = col_phenotype,
+       legend = names(col_phenotype), lty=1, lwd = 2, xpd = TRUE, ncol = 3,
+       cex = 0.7, title = "Phenotype", title.cex = 0.8)
+plot_pca(pca_res, pc_x = 3, pc_y = 4, pch = 21,
+         col = col_sample, bg = paste0(col_sample, 80),
+         labels = res$injection_index)
+```
+
+
+RUVIII
```

Comment:
I'll remove the ones for the RUV, and I need to check if I can keep the one after NOMIS